src/Compilers/CSharp/Portable/Binder/BinderFlags.cs

```diff
@@ -113,9 +113,9 @@ internal enum BinderFlags : uint
         InEEMethodBinder = 1 << 30,
 
         /// <summary>
-        /// Assume '#nullable disable' context.
+        /// Skip binding type arguments in type constraints
```

Comment:
> in type constraints [](start = 39, length = 20)

"Skip binding type arguments. For example, currently used when type constraints are bound in some scenarios."?
Also, consider describing what is used for type arguments when this flag is set.